content/participants/ttjc@ProgrammerAvinash/index.md

```diff
@@ -35,30 +35,74 @@ It would be good if someone tells me about Node JS , as i want to learn Node
 
 ### projects
 
-not a lot of projects are done. I will keep updating it with real links when done.
-
 My projects:
 
-#### Coming Soon
+#### Tetris Game
+
+_description_ the fun little game made with love 
+_stack_ Made this game in JavaScript
+_hosted link_ https://programmeravinash.github.io/TetrisGame/index.html
+
+_github link_ https://github.com/ProgrammerAvinash/TetrisGame
+
```

Comment:
Put "two spaces" after each line or add a line in between the are collapsing.
![image](https://user-images.githubusercontent.com/33570551/86215736-922cf900-bb9a-11ea-8155-13827ad18601.png)